### ISSUE TITLE
Any chance for this to be updated for TShock Mintaka? Also a feature request

### ISSUE THREAD

**Issue by Kojirremer** [ancientgods/Trivia#1]
Previously it worked with ticks, but since Mintaka has been officially introduced that's no longer an option. Could you please update this for Mintaka, kind sir?
Also, being able to customize the "interface" messages would be nice, since non-english speakers tend to not understand how to answer unless someone else tells them.

**Comment by LaurensDeV** [comment]
I updated it for you, you can find the binary here: https://github.com/ancientgods/Trivia/raw/master/Trivia/bin/Debug/Trivia.dll (I will most likely not update it on tshock because I simply do not care)

**Comment by Kojirremer** [comment]
Thanks a lot. The updated DLL started without issues (haven't yet tried answering a trivia). The .lang file caused errors if modified in any way, but just having this work again is enough.

**Comment by Kojirremer** [comment]
Sure, here it is.
[code]
[01-05-17 17:26:25] [Server API] Error: Startup aborted due to an exception in the Server API initialization:
System.InvalidOperationException: Plugin "Trivia" has thrown an exception during initialization. ---> System.IO.IOException: El proceso no puede obtener acceso al archivo 'C:\Users\Esteban\Desktop\Respaldo 05-01-17\tshock\Trivia.lang' porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   en System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   en System.IO.File.Open(String path, FileMode mode)
   en Trivia.Lang.Read()
   en Trivia.Lang..ctor()
   en CreateTrivia.Lang()
   en Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateNewObject(JsonReader reader, JsonObjectContract objectContract, JsonProperty containerMember, JsonProperty containerProperty, String id, Boolean& createdFromNonDefaultCreator)
   en Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   en Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   en Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   en Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   en Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   en Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   en Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value)
   en Trivia.Lang.Read()
   en Trivia.Lang..ctor()
   en Trivia.TriviaManager.Load_Config()
   en Trivia.TriviaManager.Initialize()
   en Trivia.Trivia.Initialize()
   en TerrariaApi.Server.ServerApi.LoadPlugins()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en TerrariaApi.Server.ServerApi.LoadPlugins()
   en TerrariaApi.Server.ServerApi.Initialize(String[] commandLineArgs, Main game)
   en TerrariaApi.Server.HookManager.InitialiseAPI()
[/code]
The file was obviously not opened by anything else at the time of the error. This only triggers if I modified the lang file, leaving it with its default value gives no error.
Ah, and I ran the file through a JSON validator just to be on the safe side and it was cleared as valid. Just in case here is how I modified it.
[code]
{
  "TimeUpMessage": "[Trivia] Se acabo el tiempo!",
  "AnnounceQuestion": "[Trivia] Responde escribiendo /a <respuesta>"
}
[/code]
